### ISSUE TITLE
SignUp/ LogIn -UI

### ISSUE THREAD

**Issue by Tushardeepak** [mann27/News24x7-Client#88]
Those pages UI are not matching with the other page UI.

**Comment by Tushardeepak** [comment]
![Screenshot (508)](https://user-images.githubusercontent.com/39951979/97975344-01d41e80-1def-11eb-91c8-2c758d622634.png)
![Screenshot (507)](https://user-images.githubusercontent.com/39951979/97975358-0567a580-1def-11eb-9446-2ea114e605da.png)
![Screenshot (506)](https://user-images.githubusercontent.com/39951979/97975364-08629600-1def-11eb-813b-55c012b86e6f.png)



Here, I have declared two new states, ( aavatar, aavatarPassword ) with initial values of true and false respectively.
I have also declared three functions which change the state of these two variables, These functions are called onClick somewhere in the code.
In img tag I have made a conditional statement as shown.

Now, when I reload the page according to the initial condition (which is aavatar: true , aavatarPassword: false )
img tag will the defaultPicLook value, but it takes the value of defaultPic which will come when ( aavatar: false, aavatarPassword: false)

I dont use class components whats the problem in this or how to achieve this,  or I have to do useEffect kind of thing how to do that in class component
